api/room_resource/schema.py

```diff
@@ -79,6 +79,31 @@ def mutate(self, info, **kwargs):
         return AssignResource(room_resource=room_resources)
 
 
+class DeleteAssignedResource(graphene.Mutation):
+    """
+        Delete assigned resource in a room
+    """
+    class Arguments:
+        resource_id = graphene.Int(required=True)
+        room_id = graphene.Int(required=True)
+    room_resource = graphene.Field(RoomResource)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, resource_id, room_id):
+        query = RoomResource.get_query(info)
+        room_resource = ResourceModel.query.filter_by(
+            id=resource_id).first()
+        assigned_resource = query.filter(
+            RoomResourceModel.room_id == room_id,
+            RoomResourceModel.resource_id == resource_id).first()
+        if not assigned_resource:
+            raise GraphQLError('Resource does not exist in the room')
+        room_resource.quantity = room_resource.quantity + assigned_resource.quantity  # noqa
```

Comment:
Good work @jesseinit . You can get rid of the `noqa` here like this `room_resource.quantity = (
            room_resource.quantity + assigned_resource.quantity)`.